samples/samples/pg_snippets.py

```diff
@@ -1364,92 +1461,90 @@ def query_data_with_query_options(instance_id, database_id):
-                          help=read_write_transaction.__doc__)
-    subparsers.add_parser("read_only_transaction",
-                          help=read_only_transaction.__doc__)
+    subparsers.add_parser("read_write_transaction", help=read_write_transaction.__doc__)
+    subparsers.add_parser("read_only_transaction", help=read_only_transaction.__doc__)
     subparsers.add_parser("add_index", help=add_index.__doc__)
-    subparsers.add_parser("read_data_with_index",
-                          help=read_data_with_index.__doc__)
+    subparsers.add_parser("read_data_with_index", help=read_data_with_index.__doc__)
     subparsers.add_parser("add_storing_index", help=add_storing_index.__doc__)
-    subparsers.add_parser("read_data_with_storing_index",
-                          help=read_data_with_storing_index.__doc__)
     subparsers.add_parser(
-      "create_table_with_timestamp", help=create_table_with_timestamp.__doc__
+        "read_data_with_storing_index", help=read_data_with_storing_index.__doc__
+    )
+    subparsers.add_parser(
+        "create_table_with_timestamp", help=create_table_with_timestamp.__doc__
     )
     subparsers.add_parser(
-      "insert_data_with_timestamp", help=insert_data_with_timestamp.__doc__
+        "insert_data_with_timestamp", help=insert_data_with_timestamp.__doc__
     )
-    subparsers.add_parser("add_timestamp_column",
-                          help=add_timestamp_column.__doc__)
+    subparsers.add_parser("add_timestamp_column", help=add_timestamp_column.__doc__)
     subparsers.add_parser(
-      "update_data_with_timestamp", help=update_data_with_timestamp.__doc__
+        "update_data_with_timestamp", help=update_data_with_timestamp.__doc__
     )
     subparsers.add_parser(
-      "query_data_with_timestamp", help=query_data_with_timestamp.__doc__
+        "query_data_with_timestamp", help=query_data_with_timestamp.__doc__
     )
-    subparsers.add_parser("insert_data_with_dml",
-                          help=insert_data_with_dml.__doc__)
-    subparsers.add_parser("update_data_with_dml",
-                          help=update_data_with_dml.__doc__)
-    subparsers.add_parser("delete_data_with_dml",
-                          help=delete_data_with_dml.__doc__)
+    subparsers.add_parser("insert_data_with_dml", help=insert_data_with_dml.__doc__)
+    subparsers.add_parser("update_data_with_dml", help=update_data_with_dml.__doc__)
+    subparsers.add_parser("delete_data_with_dml", help=delete_data_with_dml.__doc__)
     subparsers.add_parser(
-      "dml_write_read_transaction", help=dml_write_read_transaction.__doc__
+        "dml_write_read_transaction", help=dml_write_read_transaction.__doc__
     )
     subparsers.add_parser("insert_with_dml", help=insert_with_dml.__doc__)
     subparsers.add_parser(
-      "query_data_with_parameter", help=query_data_with_parameter.__doc__
+        "query_data_with_parameter", help=query_data_with_parameter.__doc__
```

Comment:
CLIs for samples are strongly discouraged: https://googlecloudplatform.github.io/samples-style-guide/#clients

If we are adding new samples it would be better to put them in their own file and follow the existing best practices